pegasus/sites.v3/code.org/views/hourofcode_sidebar.haml

```diff
@@ -0,0 +1,35 @@
+:ruby
+  corporate_donors = [{name_s: 'Amazon', url_s: 'http://www.amazon.com'}, {name_s: 'Microsoft', url_s: 'http://www.microsoft.com'}, {name_s: 'Google', url_s: 'http://www.google.com'}, {name_s: 'Juniper Networks Foundation Fund', url_s: 'http://www.juniper.net'}, {name_s: 'Salesforce', url_s: 'http://www.salesforce.com'}, {name_s: 'LinkedIn', url_s: 'http://www.linkedin.com'}, {name_s: 'JP Morgan Chase & Co', url_s: 'http://www.jpmorganchase.com/'}, {name_s: 'Allen & Co', url_s: ''}, {name_s: 'Best Buy', url_s: 'http://www.bestbuy.com/'}]
```

Comment:
pegasus/data seems like the best fit, but that's used only for gsheet-ingested data thus far, and might be confusing to have it alongside them.  Unless maybe we make something like a pegasus/data/static folder?

Otherwise alongside the .haml seems reasonable so long as the server doesn't attempt to serve that file if the user knows the URL for it.